examples/nirfsa/getting-started-iq.py

```diff
@@ -148,11 +148,14 @@ def raise_if_initialization_error(response):
     error_message = rpc_error.details()
     if rpc_error.code() == grpc.StatusCode.UNAVAILABLE:
         error_message = f"Failed to connect to server on {SERVER_ADDRESS}:{SERVER_PORT}"
+
     elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
         error_message = (
             "The operation is not implemented or is not supported/enabled in this service"
         )
+    elif rpc_error.code() == grpc.StatusCode.UNKNOWN:
+        error_message = ("Invalid Input: " + RESOURCE)
```

Comment:
You need to follow the pattern for what I did [in my PR](https://github.com/ni/grpc-device/pull/709/files). See lines 213-218 specifically.